### ISSUE TITLE
Doesn't work for MSSQL older than 2017

### ISSUE THREAD

**Issue by tjdavis3** [k1LoW/tbls#306]
### What happened

`mssql: The function 'STRING_AGG' may not have a WITHIN GROUP clause.`

### What you expected to happened

Document the database

### What stack trace or error message from tbls did you see?

mssql: The function 'STRING_AGG' may not have a WITHIN GROUP clause.

### Anything else we need to know?

STRING_AGG() first appeared in SQL Server 2017 so the use of that function prevents inspection of any but the latest versions of MSSQL.

### Environment

- `tbls version`  1.50.0
- Database version 
       Microsoft SQL Server 2012 (SP2-CU16) (KB3205054) - 11.0.5678.0 (X64) 
	Dec 16 2016 15:17:21 
	Copyright (c) Microsoft Corporation
	Enterprise Edition: Core-based Licensing (64-bit) on Windows NT 6.1 <X64> (Build 7601: Service Pack 1) (Hypervisor)

- OS ( `Ubuntu bionic`, `macOS Calalina`, `WSL2`, etc ) macOS Calalina
- Shell ( `bash`, `zsh`, `poworshell`, etc ) bash
- Stacktrace or error message from tbls ( If you can, set `DEBUG=1`)
  ```
$ DEBUG=1 tbls doc
mssql: The function 'STRING_AGG' may not have a WITHIN GROUP clause.
github.com/k1LoW/tbls/drivers/mssql.(*Mssql).Analyze
	/Users/k1low/src/github.com/k1LoW/tbls/drivers/mssql/mssql.go:175
github.com/k1LoW/tbls/datasource.Analyze
	/Users/k1low/src/github.com/k1LoW/tbls/datasource/datasource.go:113
github.com/k1LoW/tbls/cmd.glob..func5
	/Users/k1low/src/github.com/k1LoW/tbls/cmd/doc.go:71
github.com/spf13/cobra.(*Command).execute
	/Users/k1low/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850
github.com/spf13/cobra.(*Command).ExecuteC
	/Users/k1low/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958
github.com/spf13/cobra.(*Command).Execute
	/Users/k1low/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/k1LoW/tbls/cmd.Execute
	/Users/k1low/src/github.com/k1LoW/tbls/cmd/root.go:181
main.main
	/Users/k1low/src/github.com/k1LoW/tbls/main.go:34
runtime.main
	/Users/k1low/.anyenv/envs/goenv/versions/1.15.3/src/runtime/proc.go:204
runtime.goexit
	/Users/k1low/.anyenv/envs/goenv/versions/1.15.3/src/runtime/asm_amd64.s:1374
```
- Config ( .tbls.yml )
```
name: CV2
desc: Live Database
dsn: sqlserver://sa:xxxxxxxxxxxxxxxxxx@yyyyyyyyyyy.zzzzz.com:1433/LIVE
```

**Comment by k1LoW** [comment]
Hi @tjdavis3. Thank you for your report!
Unfortunately, tbls does not support MSSQL older than 2017.
Because the old MSSQL Docker image is not provided.

https://hub.docker.com/_/microsoft-mssql-server

**Comment by k1LoW** [comment]
Released as v1.68.2
